packages/critters-webpack-plugin/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "critters-webpack-plugin",
+  "version": "3.0.0",
+  "description": "Webpack plugin to inline critical CSS and lazy-load the rest.",
+  "main": "dist/critters.js",
+  "source": "src/index.js",
+  "license": "Apache-2.0",
+  "author": "The Chromium Authors",
+  "contributors": [
+    {
+      "name": "Jason Miller",
+      "email": "developit@google.com"
+    },
+    {
+      "name": "Janicklas Ralph",
+      "email": "janicklas@google.com"
+    }
+  ],
+  "files": [
+    "src",
+    "dist"
+  ],
+  "keywords": [
+    "critical css",
+    "inline css",
+    "critical",
+    "critters",
+    "webpack plugin",
+    "performance"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleChromeLabs/critters",
+    "directory": "packages/critters-webpack-plugin"
+  },
+  "scripts": {
+    "build": "microbundle --no-compress",
```

I think this alias setup should fix the dependency linkage:

```suggestion
    "build": "microbundle --target node --alias critters=../../src/index.js",
```